components/salsa-macros/src/input.rs

```diff
@@ -64,6 +64,8 @@ impl crate::options::AllowedOptions for InputStruct {
     const ID: bool = false;
 
     const REVISIONS: bool = false;
+
+    const HEAP_SIZE: bool = false;
```

Comment:
Do you plan on adding `HEAP_SIZE` for salsa-structs (input, tracked, interned) too (in a follow up PR?)

Reply:
Yeah, as we discussed, this is a little more difficult because it requires a `db` parameter. I'll do this in a future PR.